### DISCUSSION TITLE
be able to limit usage per API token

### DISCUSSION THREAD

**Discussion by capaj** [deepgram/community#890] [Feature Requests]
as per title.
I have a B2B app where I would ideally like to bill usage to my users.
Currently I've made a cloudflare worker websocket proxy which rejects if the user went over their allowed usage amount.
It would have saved me the work on this proxy if I was able to limit usage per each API token.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
